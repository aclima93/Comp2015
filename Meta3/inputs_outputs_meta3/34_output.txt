Line 10, col 19: Incompatible type in while statement (got _integer_, expected _boolean_)
