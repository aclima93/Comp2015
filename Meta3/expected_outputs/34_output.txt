Line 10, col 20: Incompatible type in while statement (got _integer_, expected _boolean_)
